src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -33,37 +33,35 @@ namespace NUnit.Engine.Runners
     /// <summary>
     /// ProcessRunner loads and runs a set of tests in a single agent process.
     /// </summary>
-    public class ProcessRunner : AbstractTestRunner
+    public partial class ProcessRunner : AbstractTestRunner
```

Comment:
partial?

Reply:
AgentProcess.CreateStartInfo.cs was ProcessRunner.CreateStartInfo.cs for a few commits. Thanks for catching!